f2/src/components/formik/formikListener/index.js

```diff
@@ -0,0 +1,21 @@
+import { useFormikContext } from 'formik'
+
+export const FormikListener = () => {
+  const { values, status, setStatus } = useFormikContext()
+  const formError = status.errors
+  let fieldsUpdated = false
+
+  if (formError) {
+    if (status.errorValues) {
+      fieldsUpdated =
+        JSON.stringify(values) !== JSON.stringify(status.errorValues)
+    } else {
+      setStatus({ errors: formError, errorValues: values })
+    }
+
+    if (fieldsUpdated) {
```

Comment:
Is this variable necessary?

Reply:
import { useFormikContext } from 'formik'

export const FormikListener = () => {
  const { values, status, setStatus } = useFormikContext()
  const formError = status.errors
  let fieldsUpdated = false

  if (formError) {
    if (status.errorValues) {
      if(JSON.stringify(values) !== JSON.stringify(status.errorValues))
        setStatus({ errors: '' })
    } else {
      setStatus({ errors: formError, errorValues: values })
    }

  }
  return null
}